### DISCUSSION TITLE
Thank you for being amazing contributor 🙏🏼

### DISCUSSION THREAD

**Discussion by hossainemruz** [hugo-themes/toha#961] [General]
I am really happy to inform you that this theme has crossed 1k GitHub stars! This was only possible because of our amazing contributors and the users. Let's together make this theme one of the best Hugo theme out there.

My deepest gratitude to:
@BernatBC , @blueyetisoftware-jason , @stueja , @jbleduigou , @Trepix , @MarionMoseby , @Rihoj , @Chow-Ray , @Kaligule , @chutommy , @vlebert , @vlebert-tactis , @noritakaIzumi , @GabrielJean , @Kyserbyte , @LNA-DEV , @Emily-0616 , @lgatellier , @alteholz , @Roxedus , @SrNetoChan , @HenzelMoras , @davidgs , @ranrinc , @daviddavo , @dilekuzulmez , @reyhhan , @aq1018 , @ormissia , @ChroniKom , @eishundo , @donfiguerres , @herzogmedia , @cmarquezrusso , @deining , @FunctionalHacker , @Nirzak , @kemalkolcuoglu , @hiroyaonoe , @mtruel , @niltied , @skamensky , @JCasahuga , @maiani , @andrewmolyuk , @llaniewski , @Cyb3r-Jak3 , @al-haras , @BenjaminAdam , @TheDarthMole , @akifmt , @zicklam , @igorr0driguez , @muhrifqii , @dietervansteenwegen , @uh-zz , @SilverLuke , @Mariatta , @andrMollo , @joyao , @mmzeynalli , @linzeyan , @will-hou , @gilcrest , @toniop99 , @serverfriendsorg , @jcabak , @priyanshsaxena , @chusopr , @Juhong-Namgung , @santiagon610 , @sarru1291 , @hblunck , @patmagauran , @vutruong99 , @apreshill , @driukova , @domenicovistocco , @hubitor, @dadosch , @ackbyte , @alex-bezek , @vincentDcmps , @freezingDaniel , @EliseBriers , @michmich112 , @abougouffa , @clarencetw , @mjasion , @salastro , @jakob42 , @ivankatliarchuk , @banana-soldier , @tkepassport39, @HabibRh26, @sylvesterroos, @Brugui7, @shavidissa, @JOSHUAJEBARAJ

### Looking for Maintainer

To make sure the theme can thrive regardless my availability, I would like to add 2-3 contributors as maintainer for this theme. If you are interested, please let me know by commenting on this thread.

Just to make it clear that I am not going anywhere or abandoning this theme. I just want to make sure that the users have a good experience, the bugs are fixed quickly, new features are added, and PRs are reviewed. Recently, the activity on this theme has slowed down and the PRs are taking long to review. I want to improve the situation.

Thank you again for being amazing contributor. 🙏🏼

**Comment by hossainemruz** [comment]
Mentioning again as GitHub did not render theses username. Thank you for your contribution.

@akifmt , @zicklam , @igorr0driguez , @muhrifqii , @dietervansteenwegen , @uh-zz , @SilverLuke , @Mariatta , @andrMollo , @joyao , @mmzeynalli , @linzeyan , @will-hou , @gilcrest , @toniop99 , @serverfriendsorg , @jcabak , @priyanshsaxena , @chusopr , @Juhong-Namgung , @santiagon610 , @sarru1291 , @hblunck , @patmagauran , @vutruong99 , @apreshill , @driukova , @domenicovistocco , @hubitor, @dadosch , @ackbyte , @alex-bezek , @vincentDcmps , @freezingDaniel , @EliseBriers , @michmich112 , @abougouffa , @clarencetw , @mjasion , @salastro , @jakob42 , @ivankatliarchuk , @banana-soldier , @tkepassport39, @HabibRh26, @sylvesterroos, @Brugui7, @shavidissa, @JOSHUAJEBARAJ

**Comment by BernatBC** [comment]
Hi @hossainemruz, on behalf of the community we would like to thank you for the incredible theme that you've developed during these years. Even if I don't have as much free time as before, I can volunteer to become a mantainer of the hugo-toha project.

**Reply by hossainemruz** [reply]
Thank you @BernatBC! I have sent you an invitation.

**Comment by Brugui7** [comment]
Hi @hossainemruz I want to thank you for creating this amazing theme, I've been using it for years and I hope there are many more to come. And, of course, I'll be contributing again! :heart:

**Reply by hossainemruz** [reply]
Thank you!

**Comment by niltied** [comment]
Hi @hossainemruz, I want to thank you too for your amazing theme. I'm happy to have contributed to this project by translating almost all of the documentation into French. Your project is fantastic, and I thought it was important to make it accessible to French-speaking people.

I don't think I'll have enough availability to be very present on the repo. But I check in from time to time to see what's going on. As I'm not a professional developer, I don't feel comfortable answering people's issues, which is why I don't feel like becoming a maintainer. @bernaBC would be a great profile for that :)

**Comment by joyao** [comment]
Hi @hossainemruz, thank you for sharing such a useful theme. I was a technical lead and full-stack web developer and am interested in becoming a maintainer of this project, even though I haven't done web development in a few years and have just started contributing to this project. I'd like to share my ideas on this project and make it more stable and better.

**Reply by hossainemruz** [reply]
Thank you @joyao! I have sent you an invitation to the organization.
